### PR TITLE
[86bze05tk][input-tags] a11y enhance

### PR DESCRIPTION
**Pull request by msereniti**
## Motivation and Context

This PR fixes a11y of InputTags component and also contains a several fixes for Tag component.

## How has this been tested?

Manually with VO and with existing tests.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [x] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Review by ilyabrower — COMMENTED** [review]
The code is ok, but check the regenerated snapshots

**Review by sheila-semrush — CHANGES_REQUESTED** [review]
Found several medium/minor issues, DMd in Slack.

**Review by sheila-semrush — CHANGES_REQUESTED** [review]
Found 2 issues:

1. Addon has incorrect position:
image

#

2. Circle is invisible:
image

**Review by sheila-semrush — APPROVED** [review]
From my side everything is ok.
@j-mnizhek can you review the docs?
This component doesn't have its own a11y props, everything comes from the children components. So I just updated the keyboard interaction table, and fixed some things in the Design guide.

**Review by j-mnizhek — APPROVED** [review]
Looks good!

I only updated content in our examples a little bit, updated and added placeholders. And also used the same text for the Accessibility documentation as we use for the Input component.

File: semcore/input-tags/src/InputTags.tsx [hunk 1]
```diff
@@ -252,32 +266,59 @@ class Value extends Component<IInputTagsValueProps> {
...
+}
+function InputTagContainerTag(props: any) {
+  const STag = Root;
+  const { getI18nText } = props;
 
   return sstyled(props.styles)(
-    <STag data-value={props.value} render={Tag} onKeyDown={onKeyDown} />,
+    <>
+      {true && (
+        <Portal>
```
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/input-tags/src/InputTags.tsx:302]: I need to place this node somewhere, to apply description (aria-descibedby works, aria-description doesn't). I don't want to put this description as node before the place it's applied

File: semcore/tag/src/style/tag.shadow.css [hunk 2]
```diff
@@ -53,6 +99,10 @@ STagContainerClose {
     text-decoration: none;
   }
 }
+/* 
+STagContainer SAddon {
+  border: 0;
+} */
```
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/tag/src/style/tag.shadow.css]: Removed

File: semcore/tag/src/Tag.jsx [hunk 3]
```diff
@@ -283,14 +330,20 @@ function Close(props) {
...
+
+  return sstyled(styles)(<SAddon render={Box} tag='div' tag-color={tagColor} />);
 }
 
 function Circle(props) {
   const SCircle = Root;
-  const { styles } = props;
-  return sstyled(styles)(<SCircle render={Box} tag='span' />);
+  const { styles, color, resolveColor } = props;
+  return sstyled(styles)(<SCircle render={Box} tag='span' tag-color={resolveColor(color)} />);
```
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/tag/src/Tag.jsx]: Added

File: semcore/input-tags/__tests__/__image_snapshots__/inputtags-renders-different-sizes.png
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/input-tags/__tests__/__image_snapshots__/inputtags-renders-different-sizes.png:1]: Fixed

File: semcore/input-tags/src/InputTags.tsx [hunk 4]
```diff
@@ -252,32 +266,59 @@ class Value extends Component<IInputTagsValueProps> {
...
 
-      return false;
-    }
-  };
+        return false;
+      }
+    },
+    [props.onClick],
+  );
+  const interactive = props.ediable || props.onClick;
```
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/input-tags/src/InputTags.tsx]: Fixed

File: semcore/input-tags/src/InputTags.tsx [hunk 5]
```diff
@@ -252,32 +266,59 @@ class Value extends Component<IInputTagsValueProps> {
...
+  );
+}
+function InputTagContainerTag(props: any) {
+  const STag = Root;
+  const { getI18nText } = props;
 
   return sstyled(props.styles)(
-    <STag data-value={props.value} render={Tag} onKeyDown={onKeyDown} />,
+    <>
+      {true && (
```
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/input-tags/src/InputTags.tsx]: Removed

File: semcore/tag/__tests__/__image_snapshots__/tagcontainer-should-support-active.png
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/tag/__tests__/__image_snapshots__/tagcontainer-should-support-active.png:1]: I see this one pixel change only in this test 🤷

**Comment by j-mnizhek** [comment]
Sorry, folks, I've just noticed that something strange happened with Tag's inner paddings in the playground. They have become bigger.

In this branch:
image
image

In the master branch (how it should be):
image
image

**Pull request merged**